pkg/entity/batch_job/batch_job.go

```diff
@@ -0,0 +1,98 @@
+package batch_job
+
+import (
+	"bytes"
+	"context"
+	"github.com/golang/protobuf/jsonpb"
+	"github.com/spf13/cobra"
+	"github.com/strmprivacy/api-definitions-go/v2/api/batch_jobs/v1"
+	"google.golang.org/grpc"
+	"io/ioutil"
+	"strmprivacy/strm/pkg/auth"
+	"strmprivacy/strm/pkg/common"
+	"strmprivacy/strm/pkg/util"
+)
+
+var client batch_jobs.BatchJobsServiceClient
+var apiContext context.Context
+
+func SetupClient(clientConnection *grpc.ClientConn, ctx context.Context) {
+	apiContext = ctx
+	client = batch_jobs.NewBatchJobsServiceClient(clientConnection)
+}
+
+func list() {
+	req := &batch_jobs.ListBatchJobsRequest{BillingId: auth.Auth.BillingId()}
+	response, err := client.ListBatchJobs(apiContext, req)
+	common.CliExit(err)
+
+	printer.Print(response)
+}
+
+func get(id *string, _ *cobra.Command) {
+	ref := &batch_jobs.BatchJobRef{
+		BillingId: auth.Auth.BillingId(),
+		Id:        *id,
+	}
+	req := &batch_jobs.GetBatchJobRequest{Ref: ref}
+	response, err := client.GetBatchJob(apiContext, req)
+	common.CliExit(err)
+	printer.Print(response)
+}
+
+func del(id *string) {
+	req := &batch_jobs.DeleteBatchJobRequest{
+		Ref: &batch_jobs.BatchJobRef{
+			BillingId: auth.Auth.BillingId(), Id: *id,
+		},
+	}
+	response, err := client.DeleteBatchJob(apiContext, req)
+	common.CliExit(err)
+	printer.Print(response)
+}
+
+func create(cmd *cobra.Command) {
+	flags := cmd.Flags()
+	batchJob := util.GetStringAndErr(flags, batch_jobs_configuration_flag_name)
```

Comment:
I find this name a bit confusing, since the name of the flag is just "file", I didn't expect this variable name to correspond with that flag. Also the data retrieved from the param is not the batchJob, but the file containing the batch job. Maybe rename it to batch_job_file_flag_name? (instead of configuration).
And rename the variable batchJob to batchJobFile?